test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -271,13 +278,13 @@ void canVisitCabinet() {
             output,
             equalTo(
                 """
-                    Returned: aggregate item [string]
-                    booze => Doc's Fortifying Wine
-                    equipment => frozen jeans
-                    food => frozen tofu pop
-                    pill => Breathitin&trade;
-                    potion => anti-odor cream
-                    """));
+                                Returned: aggregate item [string]
```

Comment:
this level of indentation seems clearly worse to me than the previous one? I don't understand why spotless would do this.

Reply:
if you can, I think it would be nice -- I both think it looked better before and think it would be good to have fewer trivial changes in this PR.